### ISSUE TITLE
Add Jupyter notebook examples

### ISSUE THREAD

**Issue by reiinakano** [reiinakano/scikit-plot#20]
It would be nice to have Jupyter notebooks in the "examples" folder showing the different plots as used in a Jupyter notebook. It could contain the same exact code as the examples in the .py files, but adjusted for size (Jupyter notebook plots tend to come out much smaller).

**Comment by lstmemery** [comment]
I can do this, if you'd like.

**Comment by reiinakano** [comment]
Hi @lstmemery ! Would be awesome if you could submit a PR! :)

**Comment by lstmemery** [comment]
Hey there, I'm getting this error with the most recent version of scikit-plot. Any ideas?

```
======================================================================
ERROR: test_curve_diffs (test_classifiers.TestPlotPrecisionRecall)
----------------------------------------------------------------------
Traceback (most recent call last)
  File "/usr/lib/python3.5/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.5/unittest/case.py", line 601, in run
    testMethod()
  File "/home/memery/Documents/scikit-plot/scikitplot/tests/test_classifiers.py", line 389, in test_curve_diffs
    ax_micro = clf.plot_precision_recall_curve(self.X, self.y, curves='micro')
TypeError: plot_precision_recall_curve() got an unexpected keyword argument 'curves'

                              no stdout                               
                              no stderr                               
======================================================================
ERROR: test_invalid_curve_arg (test_classifiers.TestPlotPrecisionRecall)
----------------------------------------------------------------------
Traceback (most recent call last)
  File "/usr/lib/python3.5/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.5/unittest/case.py", line 601, in run
    testMethod()
  File "/home/memery/Documents/scikit-plot/scikitplot/tests/test_classifiers.py", line 398, in test_invalid_curve_arg
    curves='zzz')
  File "/usr/lib/python3.5/unittest/case.py", line 728, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "/usr/lib/python3.5/unittest/case.py", line 177, in handle
    callable_obj(*args, **kwargs)
TypeError: plot_precision_recall_curve() got an unexpected keyword argument 'curves'

                              no stdout                               
                              no stderr                               
======================================================================
ERROR: test_curve_diffs (test_classifiers.TestPlotROCCurve)
----------------------------------------------------------------------
Traceback (most recent call last)
  File "/usr/lib/python3.5/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.5/unittest/case.py", line 601, in run
    testMethod()
  File "/home/memery/Documents/scikit-plot/scikitplot/tests/test_classifiers.py", line 257, in test_curve_diffs
    ax_macro = clf.plot_roc_curve(self.X, self.y, curves='macro')
TypeError: plot_roc_curve() got an unexpected keyword argument 'curves'

                              no stdout                               
                              no stderr                               
======================================================================
ERROR: test_invalid_curve_arg (test_classifiers.TestPlotROCCurve)
----------------------------------------------------------------------
Traceback (most recent call last)
  File "/usr/lib/python3.5/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.5/unittest/case.py", line 601, in run
    testMethod()
  File "/home/memery/Documents/scikit-plot/scikitplot/tests/test_classifiers.py", line 267, in test_invalid_curve_arg
    curves='zzz')
  File "/usr/lib/python3.5/unittest/case.py", line 728, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "/usr/lib/python3.5/unittest/case.py", line 177, in handle
    callable_obj(*args, **kwargs)
TypeError: plot_roc_curve() got an unexpected keyword argument 'curves'

                              no stdout                               
                              no stderr                               
=========================  test_plotters.py  =========================
.../usr/lib/python3/dist-packages/matplotlib/axes/_axes.py:531: UserWarning: No labelled objects found. Use label='...' kwarg on individual plots.
  warnings.warn("No labelled objects found. "
.
*******************************************************************************
Ran 47 test cases in 7.77s (10.83s CPU), 4 errors
2 modules OK (1 failed)
```

**Comment by reiinakano** [comment]
Hi @lstmemery , `curves` is a new argument in the latest version of scikit-plot. I am guessing that you have an older version of scikit-plot (probably 0.2.2) installed, so when you run the unit tests, you are using the older version. I'm rather new to unit testing, so I'm not sure what the best way to deal with this is. 

One way to solve this is to uninstall scikit-plot from your system first to force the unit tests to use the local repo of scikit-plot that you cloned. Try it out and see if it works.

**Comment by lstmemery** [comment]
Sorry for the delay, was busy with schoolwork. Here's the first one. I'm about to do the rest.
https://github.com/lstmemery/scikit-plot/blob/example-plots/examples/jupyter_notebooks/plot_confusion_matrix.ipynb

A couple things: 

- Should I include Jupyter and its dependencies in requirements.txt?
- Should I commit my .gitignore?

**Comment by reiinakano** [comment]
I've merged your PR #22 

I don't think I'll add Jupyter as a dependency as it should be the user's choice whether to use Jupyter or not :)

Thanks!
